calico_upgrade/pkg/calicoupgrade.go

```diff
@@ -19,26 +19,31 @@ import (
 	"os"
 
 	log "github.com/sirupsen/logrus"
-
 	"github.com/docopt/docopt-go"
+
 	"github.com/projectcalico/calico/calico_upgrade/pkg/commands"
 )
 
 func main() {
 	doc := `Usage:
   calico-upgrade [options] <command> [<args>...]
 
-    validate  Validate the v1 formatted data, check that it can be migrated to
-              the v3 format, and output a full report of any migrated names,
-              migration errors, or migrated name conflicts.
-    start     Start the upgrade process.  This will validate the data, pause
-              Calico networking, and migrate the data from v1 to v3 format.
-              Once the data is migrated successfully, the calico/node
-              instances can be upgraded to v3.x.
-    complete  This completes the upgrade process by un-pausing Calico
-              networking.
-    abort     This aborts the upgrade process by un-pausing Calico
-              networking.
+    dryrun    Perform a dryrun of the data migration. This validate that the
```

Comment:
suggest `dry-run`, just like git. And ...
- "Perform a dryrun ..." -> "Perform a dry-run ..."
- "This validate ..." -> "This validates ..."